Gemfile

```diff
@@ -27,14 +28,19 @@ gem 'sdoc', '~> 0.4.0',          group: :doc
 gem 'spring',        group: :development
 
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use unicorn as the app server
-# gem 'unicorn'
+gem 'bcrypt', '~> 3.1.7'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
 # Use debugger
 # gem 'debugger', group: [:development, :test]
-
+gem 'rspec-rails', group: [:development, :test]
```

Comment:
You've got more than a few things using the `:development, :test` groups. For organization, I'd make these blocks like `:production` below.